File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/rewarded/RewardedVideosDemoMenuActivity.java
package com.applovin.enterprise.apps.demoapp.ads.applovin.rewarded;

import android.content.Intent;

import com.applovin.enterprise.apps.demoapp.ads.applovin.DemoMenuActivity;
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem;

public class RewardedVideosDemoMenuActivity
        extends DemoMenuActivity
{
    @Override
    protected DemoMenuItem[] getListViewContents()
    {
        return new DemoMenuItem[] {
                new DemoMenuItem( "Basic Integration", new Intent( this, RewardedVideosActivity.class ) ),
                new DemoMenuItem( "Zone Integration", new Intent( this, RewardedVideosZoneActivity.class ) )
        };
    }
}

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/rewarded/RewardedVideosDemoMenuActivity.kt
package com.applovin.enterprise.apps.demoapp.ads.applovin.rewarded

import android.content.Intent
import com.applovin.enterprise.apps.demoapp.ads.DemoMenuActivity
import com.applovin.enterprise.apps.demoapp.data.main.DemoMenuItem

class RewardedVideosDemoMenuActivity : DemoMenuActivity()
{
    override fun getListViewContents(): Array<DemoMenuItem> = arrayOf(
            DemoMenuItem("Basic Integration", Intent(this, RewardedVideosActivity::class.java)),
            DemoMenuItem("Zone Integration", Intent(this, RewardedVideosZoneActivity::class.java))
    )
}
